gitpod-io/gitpod
# [JetBrains] Upgrade Gateway Plugin stable version to v223.7571.203

## Description

Upgrade JetBrains Gateway Plugin stable version to v223.7571.203. This is the Stable Gateway version since 2022-11-30.

<img width="420" alt="image" src="https://user-images.githubusercontent.com/418083/208652664-49346daa-f2fb-4da8-8102-1c6d9d2d1cf4.png">

Changes in this PR:
- The deletion of the files from `stable` folder.
- The move of files from `latest` to the root of the plugin codebase, as those classes are compatible with all v223 versions until the moment.
- The update of `plugin.xml` and `extensions.xml` to reflect the changes above.

## How to test

1. Ensure you have the Gateway installed from [JetBrains Toolbox App](https://www.jetbrains.com/toolbox-app/) and have it up-to-date.
2. Download the plugin build related to this branch in [Dev Versions](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway/versions/dev), and [install it on the Gateway](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk).
    - Download and check the stable plugin version in Gateway v223.7x.
    - Download and check the latest plugin version in Gateway v223.8x.
4. Create a new workspace from the Gateway (it's ok to use the pre-selected IDE and Repository) and confirm if JetBrains Client can connect to it.

## Release Notes

```release-note
NONE
```

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
